sql-plugin/src/main/scala/com/nvidia/spark/rapids/BatchWithPartitionData.scala

```diff
@@ -192,19 +194,24 @@ object BatchWithPartitionDataUtils {
   }
 
   /**
-   * Adds a single set of partition values to all rows in a ColumnarBatch.
+   * Adds a single set of partition values to all rows in a ColumnarBatch ensuring that
+   * size of batch is less than the maximum batch size.
```

Comment:
Comment states this is enforcing batch size limit but it does not.

Reply:
Replaced it with `ensuring that size of column is less than the maximum column size`